### ISSUE TITLE
Log in with github error

### ISSUE THREAD

**Issue by tsredanovic** [batnoter/batnoter#100]
**Describe the bug**
When clicking the `LOGIN WITH GITHUB` button I get the `Application error` screen with a message to check heroku logs.

**To Reproduce**
Steps to reproduce the bug:
1. Go to `https://batnoter.com/login`
2. Click on `LOGIN WITH GITHUB`
3. See error

**Expected behavior**
Successful log in.

**Screenshots**

<img width="1438" alt="Screenshot 2022-12-03 at 11 15 42" src="https://user-images.githubusercontent.com/50398085/205435951-0789b6bc-738d-425d-8208-bc23302047c2.png">

**Please provide browser information:**
 - Firefox

**Comment by thatshinji** [comment]
I get the same wrong, and my browser is Chrome

**Comment by WynterAC** [comment]
same problem on safari as well as chrome

**Comment by vivekweb2013** [comment]
I think Heroku stopped free hosting services. So the backend is down. I will look for other alternatives

**Comment by tsredanovic** [comment]
I've heard about this as an alternative for heroku free tier: https://fly.io/. I have not tried it yet but you might want to check it out.

**Comment by primalpoint** [comment]
Wait, so all content there is now in a limbo? Is there a way to get my projects back??

**Comment by tsredanovic** [comment]
@primalpoint No, all of your notes should still be accessible from your github account, inside the private `notes` repository.

**Comment by primalpoint** [comment]
wow, thanks a lot !

**Comment by tsredanovic** [comment]
@vivekweb2013 any updates on this? Im really loving batnoter and would hate to have to look for an alternative.

**Comment by ckim77** [comment]
Login error still persists, and my browser is Chrome

**Comment by tsredanovic** [comment]
Sadly, I dont think its coming back up any time soon. 
One alternative to keep using your notes is with [github web based editor](https://docs.github.com/en/codespaces/the-githubdev-web-based-editor). 
So if the repo for your notes is at `https://github.com/yourusername/notes` you need to replace `com` with `dev`: `https://github.dev/yourusername/notes`.

**Comment by z-x-x136** [comment]
> 1. https://batnoter.com/login

找到了吗？

**Comment by alirezajs** [comment]
Anybody find the solution?

**Comment by Perruer** [comment]
The batnoter.com server has been down since late 2022. Your notes are safe: they are plain Markdown files in your own GitHub repository.

I rewrote BatNoter so it no longer needs a server: Notewing, https://perruer.github.io/notewing/ (MIT, code at https://github.com/Perruer/notewing). The app talks to the GitHub API straight from the browser. You sign in with a fine-grained token that only has access to your notes repository.

Your existing BatNoter repository works as is: create the token, paste it, pick the repo. There is also a demo mode if you want to look first.

It also adds some of the things requested here: search, pasting screenshots, version history, drag and drop and offline use.

Disclosure: I'm the author of Notewing. Feedback is welcome in its issue tracker.
